src/coreclr/gcinfo/CMakeLists.txt

```diff
@@ -69,11 +69,12 @@ if (CLR_CMAKE_TARGET_ARCH_ARM64 OR CLR_CMAKE_TARGET_ARCH_ARM)
   create_gcinfo_lib(TARGET gcinfo_${TARGET_OS_NAME}_${ARCH_TARGET_NAME} OS ${TARGET_OS_NAME} ARCH ${ARCH_TARGET_NAME})
 endif()
 
-if (CLR_CMAKE_TARGET_ARCH_ARM64 OR CLR_CMAKE_TARGET_ARCH_AMD64)
+if (CLR_CMAKE_TARGET_ARCH_ARM64 OR CLR_CMAKE_TARGET_ARCH_AMD64 OR CLR_CMAKE_TARGET_ARCH_LOONGARCH64)
```

Comment:
Changes in this file too, can be moved to https://github.com/dotnet/runtime/pull/62886 and based on discussion there, these changes can be even further split. I see that `gcinfo` changes appear to be small overall.

Reply:
Thanks.
Had moved.